module/tabbed.lua

```diff
@@ -71,28 +129,27 @@ tabbed.pick = function()
     awful.spawn.easy_async_with_shell(xwininfo_cmd, function(output)
         for _, c in ipairs(client.get()) do
             if tonumber(c.window) == tonumber(output) then
-                if not client.focus.bling_tabbed and not c.bling_tabbed then
-                    tabbed.init(client.focus)
-                    tabbed.add(c, client.focus.bling_tabbed)
+                if client.focus == c then return end
```

Comment:
It's safer to compare the `window` property (it holds the X window id) here since it is just a number while the clients are objects/tables.